providers/com/researchregistry/harvester.py

```diff
@@ -0,0 +1,45 @@
+import json
+from furl import furl
+
+from share.harvest.harvester import Harvester
+
+# Built by inspecting http://www.researchregistry.com/browse-the-registry.html
+class ResearchRegistryHarvester(Harvester):
+    API_URL = 'https://us-api.knack.com/v1/scenes/scene_3/views/view_4/records'
+    HEADERS = {
+        'X-Knack-Application-Id': '54a1ac1032e4beb07e04ac2c',
```

Comment:
Even though it's not specific to us, like a normal API key? It's the ID of the Research Registry app, to differentiate from other unrelated applications built using Knack.